src/arkit.cs

```diff
@@ -278,6 +287,55 @@ interface ARPlaneAnchor {
 			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
 			get;
 		}
+
+		[iOS (11,3)]
+		[Export ("geometry", ArgumentSemantic.Strong)]
+		ARPlaneGeometry Geometry { get; }
+	}
+
+	[iOS (11,3)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARPlaneGeometry : NSSecureCoding {
+		[Export ("vertexCount")]
+		nuint VertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("vertices")]
+		IntPtr GetRawVertices ();
+
+		[Export ("textureCoordinateCount")]
+		nuint TextureCoordinateCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("textureCoordinates")]
+		IntPtr GetRawTextureCoordinates ();
+
+		[Export ("triangleCount")]
+		nuint TriangleCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("triangleIndices")]
+		IntPtr GetRawTriangleIndices ();
+
+		[Export ("boundaryVertexCount")]
+		nuint BoundaryVertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("boundaryVertices")]
+		IntPtr GetRawBoundaryVertices ();
+	}
+
+	[iOS (11,3)]
+	[BaseType (typeof(SCNGeometry))]
+	interface ARSCNPlaneGeometry {
```

Comment:
the base `SCNGeometry` class does not have a default .ctor
and since there seems to be a requirement for a metal device it seems likely this type cannot be created with `init`

Reply:
You're right, I missed that in the headers.

Reply:
Not specified in the headers but I agree with you it makes sense (it's safer to ship it with `[DisableDefaultCtor]`.